example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
+# decentralization swap fails, the SNS will restore the dapp canister(s) to
+# the fallback controllers (defined in `fallback_controller_principals`) and you
+# will need to start over.
 Swap:
+    # The minimum number of participants that must participate for the
+    # decentralization swap to succeed. This guarantees that a newly created SNS
+    # will have a minimum number of unique users (decentralization factor).
+    # If a decentralization swap's maximum target is reached, or the
+    # deadline elapses and there have not been at least `minimum_participants`,
+    # the swap will fail.
     minimum_participants: 57
 
-    minimum_icp:    500_000 tokens
-    maximum_icp: 10_000_000 tokens
+    # The minimum number of ICP that is required for a decentralization swap to
+    # succeed. This number divided by the number of SNS tokens being offered
+    # gives the reserve price of the swap, i.e., the minimum number of ICP per
+    # SNS token. If this amount is not achieved, the swap will fail.
+    minimum_icp: 100_000 tokens
 
-    minimum_participant_icp:     5 tokens
-    maximum_participant_icp: 1_000 tokens
+    # The maximum number of ICP that is targeted by this decentralization swap.
+    # If this amount is achieved with sufficient participation, the swap will
+    # succeed immediately, without waiting for the deadline. This means that
+    # a participant knows the minimum number of SNS tokens received per invested
+    # ICP. If this amount is achieved without reaching `minimum_participants`,
+    # the swap will immediately fail without waiting for the due date.
+    maximum_icp: 1_000_000 tokens
 
+    # The minimum amount of ICP that each participant must contribute
+    # to participate.
+    minimum_participant_icp:     10 tokens
+
+    # The maximum amount of ICP that each participant must contribute
+    # to participate.
+    maximum_participant_icp: 10_000 tokens
+
+    # The optional text that swap participants must confirm before they may
+    # participate in the swap. If set, must be within 1 to 1,000 characters.
     # confirmation_text: >
-    #     Do you understand the responsibilities and risks associated
-    #     with participating in this token swap?
+    #     I confirm my understanding of the responsibilities and risks
+    #     associated with participating in this token swap.
 
+    # An optional list of countries that should not participate in the
+    # decentralization swap. By default, participants from all countries
+    # are allowed.
     restricted_countries:
         - AQ  # Antarctica
 
+    # Configuration of the vesting schedule of the neuron basket, i.e., the SNS
+    # neurons that a participants will receive from a successful
+    # decentralization swap.
     VestingSchedule:
-        events: 104
-        interval: 7 d
+        # The number of events in the vesting schedule. This translates to how
+        # many neurons will be in each participant's neuron basket.
+        events: 3
```

Comment:
is this the total numbmer or wil it be this +1 as everyone also gets a neuron with zero dissolve delay?

Reply:
Unfortunately, this field represents the total number of neurons. (I would prefer it to be called something like `neuron_basket_size` or `num_neurons_per_participant`).

Anyways, for now the I'll add a remark that the first vesting event happens right after the swap succeeds.

Reply:
```
        # The number of events in the vesting schedule. This translates to how
        # many neurons will be in each participant's neuron basket. Note that
        # the first neuron in each neuron basket will have zero dissolve delay.
        # This value should thus be greater than or equal to `2`.
```